flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/OPTIONS/A320_Neo_CDU_OPTIONS_MCDU_KB.js

```diff
@@ -0,0 +1,84 @@
+class CDU_OPTIONS_MCDU_KB {
+    static ShowPage(mcdu) {
+
+        mcdu.clearDisplay();
+
+        const mcduInput = NXDataStore.get("MCDU_KB_INPUT", "NONE");
+        let storedMcduTimeout = parseInt(NXDataStore.get("CONFIG_MCDU_KB_TIMEOUT", "60"));
+        if (storedMcduTimeout === 0) {
+            storedMcduTimeout = "NONE";
+        }
+
+        let all = "*ALLOW INPUT[color]cyan";
+        let none = "*NO INPUT[color]cyan";
+
+        switch (mcduInput) {
+            case "ALL":
+                all = "ALLOW INPUT[color]green";
+                break;
+            default:
+                none = "NO INPUT[color]green";
+        }
```

Comment:
I don't see the need for a switch here...

```suggestion
        let all = "*ALLOW INPUT[color]cyan";
        let none = "*NO INPUT[color]cyan";
        
        if (mcduInput === "ALL") {
            all = "ALLOW INPUT[color]green";
        } else {
            none = "NO INPUT[color]green";
        }
```

one could even go ahead and make it const

```suggestion
        const [all, none] = mcduInput === "ALL" ? ["ALLOW INPUT[color]green", "*NO INPUT[color]cyan"] : ["*ALLOW INPUT[color]cyan", "NO INPUT[color]green"];
```

And while I am at it, I prefer using `{green}mytext{end}` over `mytext[color]green`, but in this case it makes no difference because there aren't multiple colors in one line/string.